content-learn/mastering-meshery-istio/introduction-to-service-meshes/linkerd/getting-started.mdx

```diff
@@ -0,0 +1,131 @@
+---
+doctype: "Chapter"
+chapterTitle: "Getting Started"
+description: "Meshery is the service mesh managemen plane which offers lifecycle, configuration, and performance management of service meshes and their workloads."
+videos: 6
+lectures: 8
+---
+
+import { ChapterStyle } from "../../../../src/components/Learn-Components/Chapters-Style/chapters.style.js";
+import LinkerdAdapter from "../../../../src/assets/images/learn-layer5/linkerd/linkerd-adapter.png";
+import InstallLinkerd from "../../../../src/assets/images/learn-layer5/linkerd/install-linkerd.png";
+
+<ChapterStyle>
+
+<h2 class="chapter-sub-heading">Deploy Linkerd</h2>
+
+With <a href="https://github.com/layer5io/linkerd-service-mesh-workshop/blob/master/prereq/README.md">prerequisites</a> complete, you are ready to download and deploy Linkerd resources.
+
+<h2 class="chapter-sub-heading">Steps</h2>
+
+1. [Install Linkerd](#1)
+1. [Verify install](#2)
+1. [Confirm add-ons](#3) 
+
+<h3 class="chapter-sub-heading">
+  <a name="1">Install Linkerd</a>
+</h3>
+
+Using Meshery, select `Linkerd` from the `Management` menu.
+
+<a href={LinkerdAdapter}>
+  <img src={LinkerdAdapter} />
+</a>
+
+In the Linkerd management page:
+
+1. Type `linkerd` into the namespace field.
+1. Click the (+) icon on the `Install` card and select `Latest Linkerd` to install the latest version of Linkerd.
+
+   <a href={InstallLinkerd}>
+     <img src={InstallLinkerd} align="center" />
+   </a>
+
+<h3>For manual steps go <a href="https://github.com/layer5io/linkerd-service-mesh-workshop/blob/master/lab-1/README.md#appendix">here</a></h3>
+
+<br />
+<br />
+
+<h4 class="chapter-alt-heading">
+  <a name="2">Verify install</a>
+</h4>
+
+Linkerd is deployed in the `linkerd` Kubernetes namespace. Verify that Linkerd and its components are deployed, execute the command:
+
+```sh
+linkerd check
+```
```

Comment:
Move this before the `Manual Installation` section above

Reply:
Also, these headings should use `h3` with class `chapter-sub-heading`, only the ones under `Manual installation` will have `h4`